scripts/download-plugins.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
```

Comment:
We should make it a part of `@theia/cli` and have a command like `theia plugins:download`